python/cudf/cudf/utils/filterutils.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
```

Comment:
Copyright year

Reply:
Also, should we move these functions into `ioutils`?

Reply:
Yep, that makes more sense. Moved to ioutils.